.travis.yml

```diff
@@ -22,3 +22,7 @@ install:
   - pip install -r requirements.txt --use-wheel
   - pip install $DJANGO --use-wheel
   - cd web && npm install && cd ..
+before_script:
+  - export DISPLAY=:99.0
```

Comment:
I would've thought you'd need to wrap this in quotes so the yaml parser doesn't read `:` as a map